### ISSUE TITLE
1.3.0 - QA Audit_database.php fails 

### ISSUE THREAD

**Issue by bmfmancini** [Cacti/cacti#5053]
When running audit_database.php the script is unable to login to the DB 

```
bash-4.4$ php audit_database.php --report
WARNING: Forcing protocol to  TCP  due to option specification. Please explicitly state intended protocol.
ERROR 1045 (28000): Access denied for user 'cacti'@'localhost' (using password: YES)
@ /var/www/html/cacti/docs/audit_schema.sql
FATAL: Failed Load the Audit Schema
ERROR:
```

**Comment by netniV** [comment]
I suspect that this may be due to the access denied.  If you have the same error when it connects properly, that shouldn't be an issue.

**Comment by netniV** [comment]
So, I didn't have your connection issue, but I did find a problem running this under windows so corrected that.  Double check your connection using the `cli/show_perms.php` command.

**Comment by bmfmancini** [comment]
Hmm Permissions show fine 
```
[root@cacti-master ~]# cd /var/www/html/cacti/cli/
[root@cacti-master cli]# php show_perms.php
                    ALTER   Yes                ALTER ROUTINE   Yes
                   CREATE   Yes                  CREATE ROLE   Yes
           CREATE ROUTINE   Yes            CREATE TABLESPACE   Yes
  CREATE TEMPORARY TABLES   Yes                  CREATE USER   Yes
              CREATE VIEW   Yes                       DELETE   Yes
                     DROP   Yes                    DROP ROLE   Yes
                    EVENT   Yes                      EXECUTE   Yes
                     FILE   Yes                 GRANT OPTION   Yes
                    INDEX   Yes                       INSERT   Yes
              LOCK TABLES   Yes                      PROCESS   Yes
                    PROXY   Yes                   REFERENCES   Yes
                   RELOAD   Yes           REPLICATION CLIENT   Yes
        REPLICATION SLAVE   Yes                       SELECT   Yes
           SHOW DATABASES   Yes                    SHOW VIEW   Yes
                 SHUTDOWN   Yes                        SUPER   Yes
                  TRIGGER   Yes                       UPDATE   Yes
                    USAGE   Yes
[root@cacti-master cli]#
```

```
[root@cacti-master cli]# php audit_database.php --report
WARNING: Forcing protocol to  TCP  due to option specification. Please explicitly state intended protocol.
ERROR 1045 (28000): Access denied for user 'cacti'@'localhost' (using password: YES)
@ /var/www/html/cacti/docs/audit_schema.sql
FATAL: Failed Load the Audit Schema
ERROR:
```

**Comment by netniV** [comment]
This may now be resolved with 6720292f6

**Comment by bmfmancini** [comment]
Yep that did it!
